src/docs/release/breaking-changes/dialog-border-radius.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Dialogs' Default BorderRadius
+description: The default BorderRadius of Dialog widgets is changing.
+---
+
+## Summary
+
+Instances of `Dialog`, as well as `SimpleDialog`, `AlertDialog`, and `showTimePicker`, now have
+a default shape of a `RoundedRectangleBorder` with a `BorderRadius` of 4.0 pixels. This matches the
+current specifications of Material Design. Prior to this change, the default behavior for
+`Dialog.shape`'s `BorderRadius` was 2.0 pixels.
+
+## Context
+
+Use of `Dialog`s and their associated subclasses (`SimpleDialog`, `AlertDialog`, and
+`showTimePicker`), appears slightly different as the border radius is larger. Golden file tests
+will fail if they contain a `Dialog` in the new default behavior.
```

Comment:
Am I reading this right? Golden tests will fail if the Dialog now uses a 4-pixel border? And the only way to fix this is to force the dialog to use a 2-pixel border?

Reply:
I've re-worded it. Please let me know if it is clearer.
A golden file test is a picture of what a widget is supposed to look like. I have changed what it looks like, so either the code should be updated to make it match the image again, or the image should be updated.